### PR TITLE
Add v0.02 changes

### PR DESCRIPTION
**Pull request by mwalmsley**
Adds support (minimal working examples, a guide) for calculating new representations with a trained model.

Also adds significant new features:
- Distributed training with several GPUs
- Metric logging with Weights&Biases (add your own login credentials)
- Train on color (3-band) images, not just greyscale

Also adds a critical bugfix (when loading images for direct predictions i.e. not via TFRecords, correctly normalise to the 0-1 interval expected (without documentation) by the tf.keras.experimental.preprocessing layers).

Also adds misc. minor fixes and documentation tweaks.

This code was used for the morphology tools paper (to be submitted shortly).

### CODE REVIEW

**Comment by mwalmsley** [comment]
Also adds both basic and advanced versions of a ringed galaxies dataset, to illustrate fine-tuning.
